docs/src/languages/php/_index.md

```diff
@@ -22,212 +30,110 @@ Note that from PHP 7.1, the images use the Zend Thread Safe (ZTS) version of PHP
-FFI is fully supported on Platform.sh.
+To use Composer 2.x on your project, in your app configuration, add the following [dependency](../../create-apps/app-reference.md#dependencies):
 
-Note: FFI is only intended for advanced use cases, and is rarely a net win for routine web requests.
-Use with caution.
-
-There are a few steps to leveraging FFI:
-
-1. Enable the FFI extension in `.platform.app.yaml`:
-
-    ```yaml
-    runtime:
-        extensions:
-            - ffi
-   ```
+```yaml {location=".platform.app.yaml"}
+dependencies:
+    php: 
+        composer/composer: '^2'
+```
 
-2. Specify a [preload file](#opcache-preloading) in which you can call `FFI::load()`.
-   Using `FFI::load()` in preload is considerably faster than loading the linked library on each request or script run.
+### Alternative repositories
 
-3. Ensure the library is available locally, but not in a web-accessible directory.
-   `.so` files may included in your repository, downloaded in your build hook, or compiled in your build hook.
-   If compiling C code, `gcc` is available by default.
-   If compiling Rust code, you can download the [Rust compiler in the build hook](https://doc.rust-lang.org/stable/book/ch01-01-installation.html).
+In addition to the standard `dependencies` format,
+you can specify alternative repositories for Composer to use as global dependencies.
+So you can install a forked version of a global dependency from a custom repository.
 
-4. For running FFI from the command line,
-   you need to enable the OPcache for command line scripts in addition to the preloader.
-   The standard pattern for the command would be `php -d opcache.preload="your-preload-script.php" -d opcache.enable_cli=true your-cli-script.php`.
+To install from an alternate repository,
+specify explicit `require` and `repositories` blocks:
 
-A working [FFI example](https://github.com/platformsh-examples/php-ffi) is available online for both C and Rust.
+```yaml {location=".platform.app.yaml"}
+dependencies:
+    php:
+        require:
+            "platformsh/client": "2.x-dev"
+        repositories:
+            - type: vcs
+              url: "git@github.com:platformsh/platformsh-client-php.git"
+```
 
-## Debug PHP-FPM
+That installs `platformsh/client` from the specified repository as a global dependency.
 
-If you want to inspect what's going on with PHP-FPM,
-you can install this [small CLI](https://github.com/wizaplace/php-fpm-status-cli):
+The `require` block is equivalent to the composer `require` command:
 
-```yaml
+```yaml {location=".platform.app.yaml"}
 dependencies:
     php:
-        wizaplace/php-fpm-status-cli: "^1.0"
+        "platformsh/client": "2.x-dev"
 ```
 
-Then when you are connected to your project over SSH, you can run:
-
-```shell
-$ php-fpm-status --socket=unix://$SOCKET --path=/-/status --full
-```
+The above is equivalent to `composer require platform/client 2.x-dev`.
```

Comment:
:lady_beetle: I think something got misplaced or something here. This doesn't seem related to the alternative repos and the whole section here needs reordering.